src/tree/ContainerAppItem.ts

```diff
@@ -103,7 +101,14 @@ export class ContainerAppItem implements ContainerAppsItem {
     }
 
     static async Get(context: IActionContext, subscription: AzureSubscription, resourceGroupName: string, containerAppName: string): Promise<ContainerAppModel> {
+        const credential = createSubscriptionContext(subscription).credentials;
         const client: ContainerAppsAPIClient = await createContainerAppsClient(context, subscription);
+        const gClient = await createGenericClient(context, createSubscriptionContext(subscription));
```

Comment:
I actually missed this on the first pass, but why is this gClient used here?

Reply:
Huh yeah, this is weird. I must have added this to test something and forgot to remove it...